src/schema/users.ts

```diff
@@ -1790,7 +1790,14 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         flags,
       });
 
-      await resubscribeUser(cio, ctx.userId);
+      await Promise.all([
+        resubscribeUser(cio, ctx.userId),
+        identifyUserPersonalizedDigest({
```

Comment:
Doesn't `identifyUser` already do all of it?

Reply:
it does the digest part but I would have to fetch user data here to be able to use it so to me it made more sense to create a separate helper to also be able to jjst set digest part of subscription.

Reply:
Ah ok let's start lean then :)